modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/macros/ProcessBuilderKaleoDesignerReact.macro

```diff
@@ -1075,6 +1111,18 @@ definition {
 		AssertElementNotPresent(locator1 = "ProcessBuilderKaleoDesignerReact#TRANSLATION_DROPDOWN");
 	}
 
+	macro viewTranslationLanguageNotPresentOnDropdownMenu {
+		AssertElementNotPresent(
+			key_locale = "${translationOption}",
+			locator1 = "ManageTranslations#DROPDOWN_MENU_ITEM_LANGUAGE");
+	}
+
+	macro viewTranslationLanguagePresentOnDropdownMenu {
+		AssertElementPresent(
+			key_locale = "${translationOption}",
+			locator1 = "ManageTranslations#DROPDOWN_MENU_ITEM_LANGUAGE");
+	}
```

Comment:
Those macros are not needed on this testcase, you should create macros to see if the language it's no longer present on the manage translations component/modal